containers/nginx-storage/file-transfer-endpoints.lua

```diff
@@ -0,0 +1,173 @@
+
+local file_transfer_operation = {}
```

Comment:
This should be named the same was as the module (`file_transfer_endpoints`).

Reply:
Done.